### PR TITLE
sys-fs/fuse-zip: bump to 0.7.0

### PR DESCRIPTION
**Pull request by okias**
- also fixes FUSE dep

Fixes: https://bugs.gentoo.org/673654

Signed-off-by: David Heidelberg <david@ixit.cz>

### CODE REVIEW

**Comment by okias** [comment]
Compile and run tested.

File: sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+inherit toolchain-funcs
+
+DESCRIPTION="FUSE file system to navigate, extract, create and modify ZIP archives"
+HOMEPAGE="https://bitbucket.org/agalanin/fuse-zip"
+SRC_URI="https://bitbucket.org/agalanin/${PN}/downloads/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (ConiKost) [inline sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild]: If empty, drop it.

File: sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="FUSE file system to navigate, extract, create and modify ZIP archives"
+HOMEPAGE="https://bitbucket.org/agalanin/fuse-zip"
+SRC_URI="https://bitbucket.org/agalanin/${PN}/downloads/${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (ConiKost) [inline sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild]: Have a look into the source code, license is not correct :-)

File: sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+DESCRIPTION="FUSE file system to navigate, extract, create and modify ZIP archives"
+HOMEPAGE="https://bitbucket.org/agalanin/fuse-zip"
+SRC_URI="https://bitbucket.org/agalanin/${PN}/downloads/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-libs/libzip-0.11.2:=
+	>=sys-fs/fuse-2.7:0="
```
Comment (johu) [inline sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild]: no slot operator, useless minimum version

File: sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+DESCRIPTION="FUSE file system to navigate, extract, create and modify ZIP archives"
+HOMEPAGE="https://bitbucket.org/agalanin/fuse-zip"
+SRC_URI="https://bitbucket.org/agalanin/${PN}/downloads/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-libs/libzip-0.11.2:=
```
Comment (johu) [inline sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild]: useless minimum version

File: sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	>=sys-fs/fuse-2.7:0="
+RDEPEND="${DEPEND}"
+
+RESTRICT="test"
+
+DOCS=( changelog README.md )
+
+PATCHES=( "${FILESDIR}/${P}-makefile.patch" )
```
Comment (johu) [inline sys-fs/fuse-zip/fuse-zip-0.7.0.ebuild]: duplicating patch with no good reason

**Comment by johu** [comment]
Will do this on my own

**Pull request closed without merging**